modules/ocf_mail/templates/postfix/main.cf.erb

```diff
@@ -13,15 +13,21 @@ myorigin = $mydomain
 # only forward mail (no local delivery), except for special aliases
 mydestination = int.ocf.berkeley.edu
 virtual_mailbox_domains =
-virtual_alias_domains = ocf.berkeley.edu
+virtual_alias_domains =
+    ocf.berkeley.edu,
+    mysql:/etc/postfix/vhost/mysql-alias-domains
 
 # logging
 always_bcc = ocflog
 
 # aliases
-virtual_alias_maps = hash:/etc/postfix/ocf/aliases-local,
-	proxy:ldap:/etc/postfix/ldap-aliases.cf
-alias_maps = hash:/etc/aliases, hash:/etc/aliases-group
+virtual_alias_maps =
+    hash:/etc/postfix/ocf/aliases-local,
+	proxy:ldap:/etc/postfix/ldap-aliases.cf,
+    mysql:/etc/postfix/vhost/mysql-alias-map
+alias_maps =
+    hash:/etc/aliases,
+    hash:/etc/aliases-group,
```

Comment:
ah, not intentional. good catch!

though, since it appears to work, we should probably just add trailing commas to the rest :D